modules/exploits/multi/php/car_rental_system.rb

```diff
@@ -0,0 +1,163 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Car Rental System 1.0 reverse shell',
+        'Description' => %q{
+          Car Rental System CMS 1.0, this module exploits a vulnerabilit, which allows to enter Admin page with SQL injection, and upload a PHP file and run a reverse shell.
+        },
+        'Author' =>
+          [
+            'Jyotsna Adhana and Saurav Shukla', # Discovery
```

Comment:
Can you please list each author individually?
```suggestion
            'Jyotsna Adhana', # Discovery
            'Saurav Shukla',  # Discovery
```